packages/vkui/src/components/FocusTrap/FocusTrap.tsx

```diff
@@ -84,7 +84,7 @@ export const FocusTrap = <T extends HTMLElement = HTMLElement>({
 
     recalculateFocusableNodesRef(parentNode);
 
-    if (arraysEquals(oldFocusableNodes, focusableNodesRef.current)) {
+    if (arraysEquals(oldFocusableNodes, focusableNodesRef.current) || !autoFocus) {
```

Comment:
nit: Может быть сначала проверить свойство `autoFocus`, чтобы зря массивы не сравнивать?

```suggestion
    if (!autoFocus || arraysEquals(oldFocusableNodes, focusableNodesRef.current) ) {
```